components/compliance-service/api/reporting/reporting.proto

```diff
@@ -175,6 +175,10 @@ message Control {
 	repeated Result results = 7;
 	repeated Ref refs = 8;
 	map<string, string> tags = 9;
+	map<string, TagValues> string_tags = 10;
```

Comment:
new field here b/c i couldn't squeeze the tags into the existing tags field due to the one:many relationship